Makefile.toml

```
@@ -266,9 +266,31 @@ cargo install \
 '''
 ]
 
+# We need Cargo version 1.51 or higher in order to build a workspace's transient dependency
+# during build-package
+[tasks.check-cargo-version]
+script_runner = "bash"
+script = [
+'''
+set -e
```

```suggestion
set -euo pipefail
```
Since we're relying on data from a non-final pipeline component.